packages/postgrest/lib/src/postgrest_builder.dart

```diff
@@ -317,18 +317,19 @@ class PostgrestBuilder<T, S, R> implements Future<T> {
   ///
   /// [url] may be used to update based on a different url than the current one
   Uri appendSearchParams(String key, String value, [Uri? url]) {
-    final searchParams = Map<String, dynamic>.from(_url.queryParametersAll);
+    final searchParams =
+        Map<String, dynamic>.from((url ?? _url).queryParametersAll);
```

Comment:
@Vinzent03 
On line 124 [here](https://github.com/supabase/supabase-flutter/blob/next/packages/postgrest/lib/src/postgrest_transform_builder.dart#L124), we generate a new URL that contains the `offset`. 

In the next line on 125, we are adding another parameter with the URL created on line 124, but because we were not using the `url` parameter to create the `searchParams` variable, what we did on line 124 is being ignored, therefore the offset is never set.

Reply:
Oh yeah, makes sense. I checked that code, but I think I viewed the code on the master branch by accident.